edxval/api.py

```diff
@@ -266,6 +266,22 @@ def create_or_update_transcript_preferences(course_id, **preferences):
     return TranscriptPreferenceSerializer(transcript_preference).data
 
 
+def remove_transcript_preferences(course_id):
+    """
+    Deletes course-wide transcript preferences.
+
+    Arguments:
+        course_id(str): course id
+    """
+    transcript_preference = None
+    try:
+        transcript_preference = TranscriptPreference.objects.get(course_id=course_id)
+    except ObjectDoesNotExist:
+        return
+
+    transcript_preference.delete()
```

Comment:
move this into `try` block, and remove unnecessary `transcript_preference=None` on #L276